waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -102,3 +102,10 @@ genPrismaSchema wasp = createTemplateFileDraft dstPath tmplSrcPath (Just templat
     entityToPslModelSchema entity =
       Psl.Generator.Model.generateModel $
         Psl.Ast.Model.Model (Wasp.Entity._name entity) (Wasp.Entity._pslModelBody entity)
+
+genMigrationsDir :: Wasp -> Maybe FileDraft
+genMigrationsDir wasp =
+  getMigrationsDir wasp >>= \waspMigrationsDir ->
```

Comment:
```suggestion
  (getMigrationsDir wasp) >>= \waspMigrationsDir ->
```
to make it a bit easier for us mortals :D

Reply:
Can add for clarity :) 👍🏻